src/engine/cuecontrol.cpp

```diff
@@ -14,9 +14,10 @@
 #include "vinylcontrol/defs_vinylcontrol.h"
 
 static const double CUE_MODE_MIXXX = 0.0;
-static const double CUE_MODE_PIONEER = 1.0;
-static const double CUE_MODE_DENON = 2.0;
-static const double CUE_MODE_NUMARK = 3.0;
+static const double CUE_MODE_MIXXX_NO_BLINK = 1.0;
```

Comment:
You can use 
`addItem(const QString & text, const QVariant & userData = QVariant())`
to make the combo-box index independent from the data.